build-tests/api-extractor-test-05/etc/api-extractor-test-05.api.md

```diff
@@ -0,0 +1,18 @@
+## API Report File for "api-extractor-test-05"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export namespace testJsLibrary {
+    let // @deprecated
+    MY_CONSTANT : number ;
```

Comment:
This looks odd... is this correct?

Reply:
This does look odd, but I imagine this is since the variable is deprecated and let's are capable of being multi-line this is how the markdown handles included the deprecated handler inline